pyxlsb/reader.py

```diff
@@ -156,7 +156,7 @@ def read_len(self):
     v = 0
     for i in range(4):
       byte = self._fp.read(1)
-      if byte == '':
+      if byte == b'':
```

Comment:
I just commented on the first one, but same thing here

Reply:
yeah sorry, I just accepted it. all other occurrences have been replaced now.